packages/client/src/client.ts

```diff
@@ -145,36 +183,41 @@ export class SkipClient {
   async assets(
     options: types.AssetsRequest = {},
   ): Promise<Record<string, types.Asset[]>> {
-    return this.cachingMiddleware(
-      "assets",
-      async (opts: types.AssetsRequest) => {
-        const response = await this.requestClient.get<{
-          chain_to_assets_map: Record<string, { assets: types.AssetJSON[] }>;
-        }>("/v2/fungible/assets", types.assetsRequestToJSON({ ...opts }));
-
-        return Object.entries(response.chain_to_assets_map).reduce(
-          (acc, [chainID, { assets }]) => {
-            acc[chainID] = assets.map((asset) => types.assetFromJSON(asset));
-            return acc;
-          },
-          {} as Record<string, types.Asset[]>,
-        );
+    const response = await this.requestClient.get<{
+      chain_to_assets_map: Record<string, { assets: types.AssetJSON[] }>;
+    }>("/v2/fungible/assets", types.assetsRequestToJSON({ ...options }));
+    const responseCamelCase = Object.entries(
+      response.chain_to_assets_map,
+    ).reduce(
+      (acc, [chainID, { assets }]) => {
+        acc[chainID] = assets.map((asset) => types.assetFromJSON(asset));
+        return acc;
       },
-      [options],
+      {} as Record<string, types.Asset[]>,
     );
+
+    if (
+      options.includeEvmAssets &&
+      options.includeSvmAssets &&
+      options.includeCW20Assets
+    ) {
```

Comment:
we only store if these params is true?

Reply:
oh I see, it's for avoiding diff params

Reply:
yeah, the thinking is that if you call it without one of them being true it might mean our cache is missing some data.

Reply:
there's probably a better / more sophisticated way to handle this but this is simple and should work for our use case